app/assets/stylesheets/splash.scss

```diff
@@ -0,0 +1,25 @@
+.splash_container {
```

Comment:
Rule set contains (11/10) properties<br>Selector `splash_container` should be written in lowercase with hyphens